### PR TITLE
fix: human event timestamps #170

### PR DESCRIPTION
**Pull request by nicolas-besnard**
# What does this PR do?

Refresh the human timestamp on the events list. I wasn't sure if you want the use to be in `hooks` or to stay with the component as it might not be used somewhere else

> Screenshot of the feature

I couldn't provide a screenshot, because the item is updating every minute

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] I have found someone to review this PR and pinged him

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

File: src/pages/events.tsx [hunk 1]
```diff
@@ -66,9 +66,26 @@ const listEvents = [
...
+  return {
+    renderKey,
+  };
+}
+
 export default function Events() {
   const [eventChecked, setEventChecked] = useState<EventCheck[]>([]);
   const [allEvents, setAllEvents] = useState<BaseEvent[]>([]);
+  const { renderKey } = useDynamicRenderKey();
+
```
Comment (RomainLanz) [inline src/pages/events.tsx:88]:
```suggestion
```

File: src/pages/events.tsx [hunk 2]
```diff
@@ -66,9 +66,26 @@ const listEvents = [
   },
 ];
 
+function useDynamicRenderKey(intervalInMs = 1_000 * 60) {
```
Comment (RomainLanz) [inline src/pages/events.tsx:69]: This should be externalize in `~/hooks/core`

**Review by RomainLanz — CHANGES_REQUESTED** [review]

**Comment by RomainLanz** [comment]
Great work!

We should also handle this in the dashboard `~/pages/dashboard.tsx`.

File: src/pages/events.tsx (hunk 2 above)
Comment (nicolas-besnard) [reply src/pages/events.tsx]: fixed in 8d077a1

**Review by RomainLanz — APPROVED** [review]

**Comment by WilliamTraoreee** [comment]
@nicolas-besnard I can't merge your PR because he has some conflicts on it. Can you fix that ? :D

**Comment by RomainLanz** [comment]
Thanks!

**Pull request merged**